### PR TITLE
BUGFIX: Remove plow-js from code tag plugin

### PR DESCRIPTION
**Pull request by mhsdesign**
This way we get rid of the console errors:

> Warning: Plow.js "$get" is deprecated! See: https://github.com/neos/neos-ui/issues/3425

### CODE REVIEW

**Comment by mhsdesign** [comment]
The plugin seems to work still after testing:

image

**Comment by mhsdesign** [comment]
And we must upmerge/cherry-pick this to 9.0 as well ;)

File: Resources/Private/CodeTagPlugin/src/manifest.js [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
-    render() {
-        const { formattingRule, inlineEditorOptions, i18nRegistry, tooltip, isActive, ...finalProps } = this.props;
-        return <IconButton {...finalProps} isActive={Boolean(isActive)} title={tooltip} />;
-    }
-}
+import {IconButtonComponent} from "./IconButtonComponent";
 
 const addPlugin = (Plugin, isEnabled) => (ckEditorConfiguration, options) => {
-    // we duplicate editorOptions here so it would be possible to write something like `$get('formatting.sup')`
+    // we duplicate editorOptions here so it would be possible to write smth like `$get('formatting.sup')`
```
Comment (Sebobo) [inline Resources/Private/CodeTagPlugin/src/manifest.js]: Why make the comment less readable?

**Review by Sebobo — CHANGES_REQUESTED** [review]
Adjust the comment to be normal English again, then it's fine 👍

File: Resources/Private/CodeTagPlugin/src/manifest.js (hunk 1 above)
Comment (mhsdesign) [reply Resources/Private/CodeTagPlugin/src/manifest.js]: idk i copied the adjusted code from the neos ui 😂

Comment (mhsdesign) [reply Resources/Private/CodeTagPlugin/src/manifest.js]: could we please keep it like this so its only one commit and im lazy 😂

Comment (Sebobo) [reply Resources/Private/CodeTagPlugin/src/manifest.js]: Just do an amend?

Comment (mhsdesign) [reply Resources/Private/CodeTagPlugin/src/manifest.js]: i did it ^^

**Review by Sebobo — APPROVED** [review]

**Pull request merged**